packages/kolibri-common/components/SearchFiltersPanel/AccordionSelectGroup.vue

```diff
@@ -40,15 +40,11 @@
                 :color="$themeTokens.primary"
               />
             </template>
-            <template
```

Comment:
I'm not sure if removing this template usage here was the best idea, but it seemed like using flex was more straightforward for this particular case, and easier to read, and that removing the template would give me a better level of customization from within this component (rather than overriding styles)

Reply:
I do think that removing this IconAfter was a good idea. I think the intention of this slot in KButton is different from what we are trying to achieve here :).